mlflow/server/js/src/test-utils/ReduxStoreFixtures.js

```diff
@@ -31,6 +32,22 @@ export const addExperimentToState = (state, experiment) => {
   };
 };
 
+export const addExperimentTagsToState = (state, experiment_id, tags) => {
+  const oldExperimentTags = state.entities.experimentTagsByExperimentId;
+  return {
+    ...state,
+    entities: {
+      ...state.entities,
+      experimentTagsByExperimentId: {
+        experiment_id: {
```

Comment:
Do you mean [experiment_id]? This will remove all tags for other experiments and have a key literally called`experiment_id `. I'd recommend adding a test for this, you'll find out while you run the test.

Reply:
This seems to only be called from a single test with a single experiment - I'll fix it so it works more generally!